packages/input/test/lion-input.test.js

```diff
@@ -38,4 +38,10 @@ describe('<lion-input>', () => {
     expect(el.getAttribute('type')).to.equal('foo');
     expect(el.inputElement.getAttribute('type')).to.equal('foo');
   });
+
+  it('has an attribute that can be used to set the placeholder text of the input', async () => {
+    const el = await fixture(`<lion-input placeholder="text"></lion-input>`);
+    expect(el.getAttribute('placeholder')).to.equal('text');
+    expect(el.inputElement.getAttribute('placeholder')).to.equal('text');
```

Comment:
I saw you resolved my earlier remark but I would like to suggest adding this:
```suggestion
    el.placeholder = 'foo';
    await el.updateComplete;
    expect(el.getAttribute('placeholder')).to.equal('foo');
    expect(el.inputElement.getAttribute('placeholder')).to.equal('foo');
```
To verify that it syncs to native input on-change. Similar to the test above this one.